Added 18.0.0 release notes & updated UMS Upgrade documentation (#8172)
umbraco/UmbracoDocs: 18/umbraco-engage/upgrading/migrate-from-umarketingsuite.md

---
description: >-
  How to move from uMarketingSuite to Umbraco Engage 18 by first migrating to
  Umbraco Engage 17.
---

# Migrate from uMarketingSuite

> You cannot migrate directly from uMarketingSuite to Umbraco Engage 18. uMarketingSuite runs on Umbraco 13, so the move to Umbraco Engage 18 is done in two separate stages.

Reviewer: ```suggestion
You cannot migrate directly from uMarketingSuite to Umbraco Engage 18. Because uMarketingSuite runs on Umbraco 13, the move requires two separate stages.
```
